src/tooling/migration-rrule-recurrence/migration-rrule-recurrence.js

```
@@ -76,34 +76,29 @@ async function update(collectionName, documents) {
-                process: newProcess,
-                state: newState,
-                data: card.data,
-                rRule: rRule
-            },
-            $unset: {
-                timeSpans: ""
-            }
-        };
-        // execute update query
-        await db.collection(collectionName).updateOne(queryForUpdate, newValuesForUpdate);
+        delete replacementCard.timeSpans;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)